app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=(announcement.currently_valid? ? '' : 'future')
```

Comment:
expired announcements still display, according to your controller implementation.